accounts/abi/bind/bind_test.go

```diff
@@ -1858,6 +1858,12 @@ func TestGolangBindings(t *testing.T) {
 		t.Fatalf("failed to replace binding test dependency to current source tree: %v\n%s", err, out)
 	}
 
+	tidy := exec.Command(gocmd, "mod", "tidy")
+	tidy.Dir = pkg
+	if out, err := tidy.CombinedOutput(); err != nil {
+		t.Fatalf("failed to go mod tidy: %v\n%s", err, out)
+	}
```

Comment:
@baptiste-b-pegasys I dont get why you added this part for.

Reply:
needed by go 1.16

Reply:
![image](https://user-images.githubusercontent.com/85155432/150744810-2bd80c9b-51cb-445a-bf30-0a3c97c39927.png)